tests/Authenticators/PrivateKeyAuthenticatorTest.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace GrahamCampbell\Tests\GitHub\Authenticators;
+
+use Github\Client;
+use GrahamCampbell\GitHub\Authenticators\PrivateKeyAuthenticator;
+use GrahamCampbell\Tests\GitHub\AbstractTestCase;
+use Lcobucci\JWT\Token;
+use Mockery;
+
+class PrivateKeyAuthenticatorTest extends AbstractTestCase
+{
+    public function testMakeSuccess()
+    {
+        $authenticator = $this->getAuthenticator();
+
+        $client = Mockery::mock(Client::class);
+        $client
+            ->shouldReceive('authenticate')->once()
+            ->with(Mockery::on(function ($token) {
+                $this->assertInstanceOf(Token::class, $token);
+
+                return true;
+            }), 'jwt');
+
+        $return = $authenticator->with($client)->authenticate([
+            'file'   => sprintf('%s/fixtures/key.pem', dirname(__DIR__)),
+            'issuer' => 1,
+        ]);
+
+        $this->assertInstanceOf(Client::class, $return);
+    }
+
+    /**
+     * @expectedException \InvalidArgumentException
+     * @expectedExceptionMessage You must inform a valid key file
```

Comment:
provide, not inform

Reply:
This message coming from JWT library I removed our check. :)